src/main/resources/db/changelog/changes/08-create-categories-table.yaml

```diff
@@ -0,0 +1,30 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-categories-table
+      author: polikarpov13
+      changes:
+        - createTable:
+            tableName: categories
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: description
+                  type: varchar(255)
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```

Comment:
Common practice that all files has one empty line in the end, including patch-sets